bigbang/bigbang_io.py

```diff
@@ -1,5 +1,5 @@
 """
-Input/Output for Listserv data.
+Input/Output for mailing data.
```

Comment:
If this is abstracting over mailing IO, maybe rename this file to mailing_io.py.

Bigbang supports other kind of data IO besides mail.

Reply:
I tried to clarify the purpose of `bigbang/bigbang_io.py` in the docstrings and in replies below.